openseespy-pip/openseespy/postprocessing/Get_Rendering.py

```diff
@@ -120,48 +120,48 @@ def readODB(*argv):
 	First record all the output data and then read it instantly for plotting.
 	"""
     
-	ModelName = argv[0]
```

Comment:
Are these indentation changes? It is only in readODB() and not in createODB().

Reply:
I had some issues with the tabs on my end, - the script won't run on my end unless it's all spaces or all tabs. 
It should be converted back to tabs now.